### PR TITLE
adapt pyproject.toml and delete setup files

### PR DESCRIPTION
**Pull request by DaRa42**

### CODE REVIEW

**Comment by romulogoncalves** [comment]
@DaRa42 we also need to update tox.ini

**Comment by romulogoncalves** [comment]
@DaRa42 we also need to update CONTRIBUTING.rst, there setup.py  is also used.

**Comment by DaRa42** [comment]
@romulogoncalves CONTRIBUTING.rst is updated

File: {{cookiecutter.project_slug}}/CONTRIBUTING.rst [hunk 1]
```diff
@@ -70,16 +70,14 @@ Commit Changes
...
 ~~~~~~
 
-1. Fork the `{{ cookiecutter.project_slug }}` repo on GitLab.
-2. Clone your fork locally::
+1. Clone the repository::
 
     $ git clone {{ giturl }}
 
-3. Install your local copy into a virtualenv. Assuming you have virtualenvwrapper installed, this is how you set up your fork for local development::
+2. Create an environment::
```
Comment (romulogoncalves) [inline {{cookiecutter.project_slug}}/CONTRIBUTING.rst:77]: We are jumping from 2 to 4. Should not we also install the package using `pip install .`? That should be the replacement for `python setup.py install`.

**Review by romulogoncalves — APPROVED** [review]

**Pull request merged**